code/modules/research/xenoarchaeology/artifact/artifact_gigadrill.dm

```diff
@@ -5,7 +5,6 @@
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "gigadrill"
 	var/active = 0
-	var/drill_time = 10
```

Comment:
So you're just gonna remove the variable but not remove any of the references to it?

You're not even trying are you?